edgelet/api/identityServiceVersion_2020_06_01.yaml

```diff
@@ -0,0 +1,102 @@
+swagger: '2.0'
+schemes:
+  - http
+info:
+  title: Identity Service API
+  version: '2020-02-02'
```

Comment:
The filename says `2020-06-01`, this says `2020-02-02` and the `api-version` parameter says `2020-06-28`. They should all be consistent. Same for the KS file.

Reply:
Copy pasta errors from another swagger. Thanks for catching this.